supervisor/shared/external_flash/devices.h

```diff
@@ -442,4 +442,23 @@ typedef struct {
     .write_status_register_split = false, \
     .single_status_byte = false, \
 }
+
+// Settings for the Micron N25Q128A 16MiB SPI flash.
```

Comment:
Did you get this working? If not, we shouldn't add it.

Reply:
It's for the F412. I don't expect issues but it isn't connected to an SPI line, so it can't use the regular module. I'll save it for later.